content/en/tracing/metrics/metrics_namespace.md

```diff
@@ -39,21 +39,18 @@ Trace metrics are generated for service entry spans and certain operations depen
 The [trace metrics][3] namespace is formatted as:
 
 - `trace.<SPAN_NAME>.<METRIC_SUFFIX>`
-- `trace.<SPAN_NAME>.<METRIC_SUFFIX>.<2ND_PRIM_TAG>_service`
 
 With the following definitions:
 
 `<SPAN_NAME>`
 : The name of the operation or `span.name` (examples: `redis.command`, `pylons.request`, `rails.request`, `mysql.query`).
 
 `<METRIC_SUFFIX>`
-: The name of the metric (examples: `duration`, `hits`, `span_count`). See the section below.
-
-`<2ND_PRIM_TAG>`
-: If the metric name accounts for the [second primary tag][4], this tag is part of the metric name.
+: The name of the metric (examples: `hits`, `errors`, `apdex`, `duration`). See the section below.
 
 `<TAGS>`
-: Trace metrics tags, possible tags are: `env`, `service`, `version`, `resource`, `sublayer_type`, `sublayer_service`, `http.status_code`, `http.status_class`, Datadog Agent tags (including the host and second primary tag). **Note:** Tags set on spans do not count and will not be available as tags for your traces metrics.
+: Trace metrics tags, possible tags are: `env`, `service`, `version`, `resource`, `http.status_code`, `http.status_class`, Datadog Agent tags (including the host and second primary tag). 
```

Comment:
`http.status_class` is not on the hits/errors metrics for HTTP spans. Just `http.status_code`.

Reply:
yes these are just all the possible tags on metrics but not necessarily all of them are on each. Do you read this as misleading?

Reply:
```suggestion
: Trace metrics tags, possible tags are: `env`, `service`, `version`, `resource`, `http.status_code`, `http.status_class`, and Datadog Agent tags (including the host and second primary tag). 
```